CDTDatastore/Encryption/CDTEncryptionKeySimpleProvider.h

```diff
@@ -27,12 +27,14 @@
  @see CDTEncryptionKeyProvider
  @see CDTEncryptionKey
  */
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
Again I think this audited region allows us to remove `nonnull` from `NSData *` on lines 35, 37